rueppellii-lvlup-asp.net-core.IntegrationTests/Scenarios/AdminControllerTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Newtonsoft.Json;
+using rueppellii_lvlup_asp.net_core.DTOs;
+using rueppellii_lvlup_asp.net_core.IntegrationTests.Fixtures;
+using rueppellii_lvlup_asp.net_core.IntegrationTests.Mocks;
+using rueppellii_lvlup_asp.net_core.Utility;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace rueppellii_lvlup_asp.net_core.IntegrationTests.Scenarios
+{
+    [Collection("TestCollection")]
+    public class AdminControllerTest
+    {
+        private readonly TestContext _testContext;
+
+        public AdminControllerTest(TestContext testContext)
+        {
+            this._testContext = testContext;
+        }
+        [Fact]
```

Comment:
styleguide: put an empty line before [Fact] please

Reply:
Fixed the empty lines / styleguide in the Test.